editor/editor_settings.cpp

```diff
@@ -818,8 +818,8 @@ void EditorSettings::_load_defaults(Ref<ConfigFile> p_extra_config) {
 	// Output
 	EDITOR_SETTING(Variant::INT, PROPERTY_HINT_RANGE, "run/output/font_size", 13, "8,48,1")
 	_initial_set("run/output/always_clear_output_on_play", true);
-	_initial_set("run/output/always_open_output_on_play", true);
-	_initial_set("run/output/always_close_output_on_stop", false);
+	EDITOR_SETTING(Variant::INT, PROPERTY_HINT_ENUM, "run/output/action_on_play", 0, "Open Output,Open Debugger,Do Nothing")
+	EDITOR_SETTING(Variant::INT, PROPERTY_HINT_ENUM, "run/output/action_on_stop", 1, "Close Bottom Panel,Do Nothing")
```

Comment:
Since this is no longer specific to the Output panel, we should maybe rename the category.

Maybe `run/bottom_panel/` ?

Reply:
Was about to suggest this. The 2 old output-related ones can be put into an "Output" subsection (because you woudn't know what "Font Size" refers to otherwise)

Reply:
I moved them in a new section "Bottom panel":
![image](https://github.com/godotengine/godot/assets/81109165/5c2611a0-2ac9-4a2b-b3d2-b40e33e8d4ef)